File: 백준/Bronze/2480. 주사위 세개/주사위 세개.c
#define max(x, y) ((x) > (y) ? (x) : (y))
    
int main()
{
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);

    //틀려서 다른 풀이 참고함.
    
    if(a == b && b == c) //3
        printf("%d", 10000 + a*1000);
    else if(a == b || a == c) //2
        printf("%d", 1000 + a*100);
    else if(b == c) //2
        printf("%d", 1000 + b*100);
    else //0
        printf("%d", max(max(a, b), c)*100);
    

    return 0;
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split(" ");

let [a, b, c] = input.map(Number);

// 1. 같은 눈이 3개가 나오는 경우
if (a == b && b == c) {
  console.log(10000 + a * 1000);
}
  
// 3. 모두 다른 눈이 나오는 경우 
else if (a != b && b != c && c != a) {
  console.log(Math.max(a, b, c) * 100);
}

// 2. 같은 눈이 2개만 나오는 경우
else {
  if (a == b) {
    console.log(1000 + a * 100);
  } else if (b == c) {
    console.log(1000 + b * 100);
  } else if (c == a) {
    console.log(1000 + c * 100);
  }
}